### ISSUE TITLE
can we train this model in 1024 pix?

### ISSUE THREAD

**Issue by heart-du** [XLabs-AI/x-flux#56]

**Comment by Anghellia** [comment]
Yes, we trained our LoRA and ControlNet models at 1024x1024 resolution

**Comment by heart-du** [comment]
Do you need to configure model parallelism, for example, deepspeed zero-3?

**Comment by appleyang123** [comment]
Have you trained ControlNet models at 16-9 (width=1360, height=768) resolution?
I train controlnet on 1024-1024 resolution, it works. But when training on 16-9, the results are blurred.

**Comment by Artemis1111** [comment]
@appleyang123 When training with 1024x1024 resolution, I’d like to know your learning rate (LR), the number of images in your dataset, the number of steps, and how many epochs you trained.

**Comment by okdalto** [comment]
I'm going through the same issue


```
model_name: "flux-dev"
data_config:
  train_batch_size: 1
  num_workers: 4
  img_size: 512
  img_dir: images/
  random_ratio: true # support multi crop preprocessing
report_to: wandb
train_batch_size: 1
output_dir: lora/
max_train_steps: 100
learning_rate: 1e-5
lr_scheduler: constant
lr_warmup_steps: 10
adam_beta1: 0.9
adam_beta2: 0.999
adam_weight_decay: 0.01
adam_epsilon: 1e-8
max_grad_norm: 1.0
logging_dir: logs
mixed_precision: "bf16"
checkpointing_steps: 2500
checkpoints_total_limit: 10
tracker_project_name: lora_test
resume_from_checkpoint: latest
gradient_accumulation_steps: 2
rank: 16
single_blocks: "1,2,3,4"
double_blocks: null
disable_sampling: false
sample_every: 250 # sample every this many steps
sample_width: 1024
sample_height: 1024
sample_steps: 20

```

This is my `test_lora.yaml`

**Comment by lijing1996** [comment]
try to pre-cache text embedding and vae embedding in disk
